### PR TITLE
Bryan: Created base about me files

### PR DESCRIPTION
**Pull request by bz5989**
Screenshot 2024-06-30 at 1 24 44 PM

Currently putting at ready for review because full modification of about me section would be a pretty massive commit (especially since I reorganized the files)

L=ENG-102

### CODE REVIEW

File: js/src/Routes.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { AmandaPage } from './personal/amanda/Amanda.page.tsx'
 import { BryanPage } from './personal/bryan/Bryan.page.tsx'
-import { MainPage as BryanMainPage } from './personal/bryan/Main.page.tsx'
-import { ProjectsPage as BryanProjectsPage } from './personal/bryan/Projects.page.tsx'
-import { FunPage as BryanFunPage } from './personal/bryan/Fun.page.tsx'
-import { AboutMePage as BryanAboutMePage } from './personal/bryan/AboutMe.page.tsx'
+import { MainPage as BryanMainPage } from './personal/bryan/Main/Main.page.tsx'
+import { ProjectsPage as BryanProjectsPage } from './personal/bryan/Projects/Projects.page.tsx'
+import { FunPage as BryanFunPage } from './personal/bryan/Fun/Fun.page.tsx'
+import { AboutMePage as BryanAboutMePage } from './personal/bryan/About Me/AboutMe.page.tsx'
```
Comment (HaoxinLuo) [inline js/src/Routes.tsx:6]: nit: don't have spaces in directory and file names. Also not sure if Henry suggested any convention on casing? The mix of starting with lower and upper case is a bit weird

File: js/src/personal/bryan/About Me/AboutMe.page.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Title, Group } from '@mantine/core'
+import classes from '../Bryan.module.css'
```
Comment (HaoxinLuo) [inline js/src/personal/bryan/About Me/AboutMe.page.tsx:2]: nit: try to use the absolute paths like '@/.../personal/bryan/...'

Absolute paths are easier to refactor and search for

File: js/src/personal/bryan/About Me/AboutMeText.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Text, Space } from '@mantine/core'
+import classes from '../Bryan.module.css'
```
Comment (HaoxinLuo) [inline js/src/personal/bryan/About Me/AboutMeText.tsx]: nit: worth splitting up the main css module into different pages - especially in the next PR

File: js/src/personal/bryan/Frame/NavBar.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { Container, Group, Anchor } from '@mantine/core'
-import { Settings } from './Settings.tsx'
-import classes from './Bryan.module.css'
+import { Settings } from '../Settings/Settings.tsx'
+import classes from '@/.../personal/bryan/Bryan.module.css'
```
Comment (HaoxinLuo) [inline js/src/personal/bryan/Frame/NavBar.tsx:3]: MB I meant this as an example, not sure if the `@/.../` thing works?
'@/personal/bryan/Bryan.module.css' should work though

**Review by HaoxinLuo — APPROVED** [review]

**Comment by HaoxinLuo** [comment]
Only nits so feel free to fix or ignore them and just merge. Do remember to test that your page works before it though.

File: js/src/personal/bryan/Frame/NavBar.tsx (hunk 4 above)
Comment (bz5989) [reply js/src/personal/bryan/Frame/NavBar.tsx:3]: oh yeah, tested and it didn't work 💀 
Hard resetting to previous commit XD

**Review by HaoxinLuo — DISMISSED** [review]

**Comment by HaoxinLuo** [comment]
thx for the fast responses - no expectation of checking these outside of work hours btw

**Review by HaoxinLuo — APPROVED** [review]

**Pull request merged**